main.lua

```diff
@@ -0,0 +1,104 @@
+imgxKaizer=200
+imgyKaizer=0
+
+imgxC1 = 0
+imgyC1 = 100
+
+imgxC2 = 0
+imgyC2 = 200
+
+imgxC3 = 0
+imgyC3 = 300
+
+function love.load()
+    imageKaizer = love.graphics.newImage("Kaizer.JPG")
+    widthKaizer = imageKaizer:getWidth()
+    heightKaizer = imageKaizer:getHeight()    
+    
+    imageCar1 = love.graphics.newImage("Car1.JPG")
+    widthCar1 = imageCar1:getWidth()
+    heightCar1 = imageCar1:getHeight()
+
+    imageCar2 = love.graphics.newImage("Car2.JPG")
+    widthCar2 = imageCar2:getWidth()
+    heightCar2 = imageCar2:getHeight()
+
+    imageCar3 = love.graphics.newImage("Car3.JPG")
+    widthCar3 = imageCar3:getWidth()
+    heightCar3 = imageCar3:getHeight()
+    
+
+end
+
+function love.update(dt)
+    
+    imgxC1 = imgxC1 + 300 * dt
+
+    if imgxC1 > 1000 then
+        imgxC1 = 0
+    end
+
+    imgxC2 = imgxC2 - 300 * dt
+
+    if imgxC2 < 0 then
+        imgxC2 = 1000
+    end
+
+    imgxC3 = imgxC3 + 300 * dt
+
+    if imgxC3 > 1000 then
+        imgxC3 = 0
+    end

```

Comment:
these statements are all related to the movement of cars, consider putting them into a separate function.

Reply:
Tom fixed by using a single for loop to calculate for all cars in implemented tables.